src/App.js

```diff
@@ -8,20 +8,16 @@ import UploadMats from './pages/uploadmats.js';
 import Progress from './pages/manage_progress.js';
 import StudentDashboard from './pages/student_dashboard.js';
 import TeacherDashboard from './pages/teacher_dashboard.js';
+import StudentPage from './pages/forum_student.js';
+import TeacherPage from './pages/forum_teacher.js';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <Routes>
-        <Route path='/' element={<LoginPage/>}></Route>
```

Comment:
# Changes:
- you are removing these paths.
- They are needed for the flow of the app.
- For now, in App.js we should only be adding stuff not removing. Double check app.js with the main branch.